src/main/java/org/jabref/gui/LibraryTabContainer.java

```diff
@@ -20,4 +18,6 @@ public interface LibraryTabContainer {
     void closeTab(LibraryTab libraryTab);
 
     void closeCurrentTab();
+
+    void refresh();
```

Comment:
Can you add some JavaDoc here explaining briefly the intention of this method?

Reply:
Done. In theory in the long term: this method should vanish, as everything dependend on the preferences should work with observables and refresh automatically. But there is still a road to walk.